### ISSUE TITLE
Export images/groups?

### ISSUE THREAD

**Issue by ngeorgop** [ViDA-NYU/ARIES-Issues#17]
Is there any function that will allow me to export a checklist of images from a particular group or project?

**Comment by joaorulff** [comment]
Yes. You can export a CSV of the project by clicking on the menu right beside the metadata menu. 

However, you can only export for an entire project.

Thanks

**Comment by fabwolf** [comment]
@joaorulff Hi there, thanks for this useful piece of software. Are you planning to add these functionalities, i.e. the possibility to export a csv-file of a specific group and to download this set of image files, maybe even keep the original filename with which they were uploaded?
This would open up a lot of use cases, I think...
Thanks a lot,
Fabian

**Comment by joaorulff** [comment]
Hello, Fabian

Sorry about the late reply.
We plan to add these functionalities in the future, however, there is no specified time for these features to be implemented.

Thanks for your message!

**Comment by ngeorgop** [comment]
Thanks for all of this information! This has been working great, but recently I've had an issue downloading images. I click the "Download Project (PPT)" button and nothing happens. I'm still able to download the metadata CSV though. Any ideas?

**Comment by joaorulff** [comment]
Hello @ngeorgop 

Thanks for your feedback.
Could you please confirm if you're using Google Chrome?

Thanks!

**Comment by ngeorgop** [comment]
Yes, that's correct.

**Comment by joaorulff** [comment]
Could you tell me the operating system you're using?

**Comment by ngeorgop** [comment]
Mac OS Mojave

**Comment by ngeorgop** [comment]
Hi there, following up on this issue. I am still unable to download projects as a PPT, though I can download CSVs. Is there anything I can try to fix this issue? Thank you!

**Comment by joaorulff** [comment]
Hello, 

After clicking on the "Download Project (PPT)" button in the top corner of the metadata box, are you able to download any file?
If yes, is the file coming with any problem?

Thanks for your feedback!

**Comment by ngeorgop** [comment]
Nothing happens when I click "Download Project (PPT)." I've tried relaunching my browser, signing out/in, etc - nothing has made a difference.

**Comment by joaorulff** [comment]
Ok, thanks.

If you're using Google Chrome, can you click on "Tools" > "Javascript Console" on your browser and send me a screenshot of the console after you click on "Download Project (PPT)"?

Thanks!

**Comment by ngeorgop** [comment]
![Screen Shot 2019-08-16 at 5 15 26 PM](https://user-images.githubusercontent.com/47093039/63198702-82232e00-c049-11e9-829b-063ea6222f10.png)

**Comment by ngeorgop** [comment]
i tried again, and this appeared the second time
![Screen Shot 2019-08-16 at 5 16 54 PM](https://user-images.githubusercontent.com/47093039/63198761-b991da80-c049-11e9-9091-5cdd0de8b4d0.png)

**Comment by ngeorgop** [comment]
Hello, following up on this. Is there something that I am doing wrong, based on the screenshots? Thanks so much.

**Comment by joaorulff** [comment]
Hi,

We're still investigating this issue.
We're trying to simulate a case where this tool doesn't work in order to understand what's happening in your machine. 

Thanks for your feedback!

**Comment by ngeorgop** [comment]
Hello, I'm following up on this issue that I flagged years ago. I am still unable to export PPTs of my projects, no matter what machine or browser I'm working on. I can export as a CSV, but not PPT. When I select "Download Project [PPT]" nothing happens at all. 

I am really hoping there is a fix for this, as I spent a lot of time adding images to one of my projects, but it's rather useless to me if I can't export them for use. Please let me know if there's any further information I can provide.  Thank you!

**Comment by joaorulff** [comment]
Hello, @ngeorgop 

This issue was solved when it was first opened.
For most users we are supporting the "Download Project" features works fine. Please, refer to the video attached to this message.

There's a formatting issue with the ppt due to updates on the library we are using, but you can work around this issue just by formatting the ppt in PowerPoint.

Are you still using the same configs you provided us before? If not, can you tell us the browser you're currently using and the OS? I'll try to spend more time checking it. 

https://user-images.githubusercontent.com/7430591/135934583-8c0874f7-57f0-4f14-8847-6bd20bcc30e4.mov

**Comment by ngeorgop** [comment]
Hi Joao, thanks for your response! Yes, I am still using google chrome. One thing I realized this afternoon - I can download one of my projects as a PPT, and it's much smaller than the one I'm trying to download now (6 images versus 140). Might that be the issue? That the project and its data are too large?

Thanks!
